src/review-form.ts

```diff
@@ -0,0 +1,272 @@
+        rows="10"
+        cols="50"
+        required
+      ></textarea>`;
+  }
+
+  private get actionButtonsTemplate(): HTMLTemplateResult {
+    return html`<div class="action-btns">
+      ${this.identifier
+        ? html`<a
+            class="btn cancel"
+            href=${`${this.baseHost ?? ''}/details/${this.identifier}`}
+          >
+            Cancel
+          </a>`
+        : nothing}
+
+      <button
+        type="submit"
+        class="btn submit"
+        name="submit"
+        value="Submit review"
+      >
+        Submit review
+      </button>
+    </div>`;
+  }
+
+  private renderStar(num: number): HTMLTemplateResult {
+    const isSelected = num === this.currentStars;
+    const ratingLabel = `Rate ${num > 1 ? `${num} stars` : '1 star'}`;
+
+    return html`<button
+      class=${`star star-${num}`}
+      title=${isSelected ? 'Clear rating' : `${ratingLabel}`}
+      @click=${(e: Event) => this.handleStarClicked(e, num)}
+    >
+      ${num <= this.currentStars ? starSelected : starUnselected}
+    </button> `;
+  }
+
+  /* Prevents form submission and sets stars based on number clicked */
+  private handleStarClicked(e: Event, num: number): void {
+    e.preventDefault();
+    this.setStars(num);
+  }
+
+  /* Clear stars if requested */
+  private handleClearBtnClicked(e: Event): void {
+    e.preventDefault();
+    this.currentStars = 0;
+  }
+
+  /* Updates star count, resetting to 0 if the currently selected star is clicked */
+  private setStars(num: number): void {
+    this.currentStars = num === this.currentStars ? 0 : num;
+  }
+
+  static styles = css`
+    :host {
+      font-family: var(
+        --ia-font-stack,
+        'Helvetica Neue',
+        Helvetica,
+        Arial,
+        sans-serif
+      );
+
+      color: var(--ia-text-color, #2c2c2c);
+    }
+
+    .form-heading {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      padding-top: 1.5rem;
+      font-size: 1.6rem;
+      font-weight: bold;
+    }
+
+    label {
+      display: inline-block;
+      margin-bottom: 5px;
+    }
+
+    textarea,
+    input[type='text'] {
+      padding: 0.5rem;
+      width: calc(100% - 1rem);
+      font-family: inherit;
+    }
+
+    .stars {
+      display: flex;
+      flex-direction: row;
+      gap: 0.2rem;
+      align-items: center;
+    }
+
+    .star {
+      all: unset;
+      width: 3rem;
+    }
+
+    .star:hover {
+      cursor: pointer;
+    }
+
+    .clear-stars-btn {
+      padding: 0 0.5rem;
+      color: var(--ia-link-color, #4b64ff);
+      font-family: inherit;
+      border: none;
+      background: transparent;
+      display: inline-block;
+      padding-top: 0.5rem;
+    }
+
+    .clear-stars-btn:hover {
+      cursor: pointer;
+      text-decoration: underline;
+    }
+
+    .action-btns {
+      width: 100%;
+      display: flex;
+      justify-content: flex-end;
+      gap: 1rem;
+      padding-top: 1.5rem;
+    }
+
+    .btn {
```

Comment:
Suggestion: for button styles, you can use `ia-button` style from https://github.com/internetarchive/iaux-collection-browser/blob/main/src/styles/ia-button.ts. It has all button effects we are using most of new offshoot buttons.